web/js/layers/active.js

```diff
@@ -402,9 +405,54 @@ export function layersActive(models, ui, config) {
     if (layer.palette) {
       renderLegendCanvas(layer);
     }
+    toggleSubdaily();
     sizeProductsTab();
   };
 
+  var subdailyCheck = function () {
+    var activeLayers = models.layers.active;
+    var currentProjection = models.proj.selected.id;
+    var check;
+    lodashEach(activeLayers, function(activeLayer) {
+      if (Object.keys(activeLayer.projections).some(function(k) { return ~k.indexOf(currentProjection); })) {
+        if (activeLayer.period === 'subdaily' && activeLayer.projections[currentProjection]) check = true;
+      }
+    });
+    return check;
+  };
+
+  var setMaxZoomlevel = function (zoomLevel) {
+    models.date.maxZoom = zoomLevel;
+  };
+
+  var toggleSubdaily = function () {
```

Comment:
Instead of toggling all the individual elements, I would suggest toggling a class on the parent element and doing the rest in CSS.

Reply:
👍 👍